html-tags-void.json.d.ts

```diff
@@ -1,3 +1,28 @@
-declare const voidHtmlTags: readonly string[];
+export type HTMLTagsVoid =
+	| 'area'
+	| 'base'
+	| 'br'
+	| 'col'
+	| 'embed'
+	| 'hr'
+	| 'img'
+	| 'input'
+	| 'link'
+	| 'menuitem'
+	| 'meta'
+	| 'param'
+	| 'source'
+	| 'track'
+	| 'wbr'
 
-export = voidHtmlTags;
+/**
+List of standard, self-closing HTML tags.
+@example
+```
+import htmlTags = require('html-tags/html-tags-void.json');
+console.log(htmlTags);
+//=> ['a' | 'abbr' | 'acronym' …]
+```
+*/
+declare const htmlTags: readonly HTMLTagsVoid[];
+export default htmlTags;
```

Comment:
You cannot use `export default`. This package is not (and cannot yet be) ESM.

Here's how to do it: https://github.com/sindresorhus/typescript-definition-style-guide/tree/57a925bb5b3eb46a4666eab31e1c883a26b7bfb5#documentation

Reply:
Should I fix https://github.com/sindresorhus/html-tags/blob/main/index.d.ts also, even though it's been merged?

Reply:
Also, the tests require two exports - a type, and an array. As far as I know, I can't export both in CommonJS.

Reply:
> Should I fix index.d.ts as well?

Yes, that would be great.

> Also, the tests require two exports - a type, and an array. As far as I know, I can't export both in CommonJS.

I'm not clear what you're referring to, but the link I shared shows how to do it. You need to use a `namespace`.

Reply:
> You need to use a `namespace`.

The type inside it would have the same name as the namespace. Is that okay?
```ts
// File: html-tags-void.json.d.ts
declare namespace htmlTagsVoid {
	type htmlTagsVoid =
		| 'area'
		| 'base'
		// ...
}

/** ... */
declare const htmlTagsVoid: readonly htmlTagsVoid.htmlTagsVoid[];

export = htmlTagsVoid;
```

Reply:
That's fine. This is all just temporary anyway. It will be much nicer when we can use ESM.